powershell/plugins/plugin-create-inline-properties.ts

```diff
@@ -43,6 +44,16 @@ function getNameOptions(typeName: string, components: Array<string>) {
   return [...result.values()];
 }
 
+function getCombinedDescription(rawDescription: string, externalDocsUrl?: string, externalDocsDescription?: string): string {
+  let description = rawDescription ?? "";
+  if (undefined !== externalDocsUrl && "" !== externalDocsUrl) {
+    description = description.concat(" Please visit external url ", externalDocsUrl, " to get more information.")
+    if (undefined !== externalDocsDescription && "" !== externalDocsDescription) {
+      description = description.concat(" The content of external url is about '", externalDocsDescription, " '.")
```

Comment:
Suggest removing it for the time being. It seems a little strange to tell users of the cmdmlet something like `The content of external url is about xxxx`

Reply:
Do you mean we ignore the description filed for the time being? This description is possible for parameters or properties.